items.json

```diff
@@ -30698,14 +30698,14 @@
     "weapons": []
   },
   {
-    "id": "crpg_byzantine_mace_v1_h0",
-    "baseId": "crpg_byzantine_mace_v1",
+    "id": "crpg_byzantine_mace_v2_h0",
+    "baseId": "crpg_byzantine_mace_v2",
     "name": "Byzantine Mace",
     "culture": "Empire",
     "type": "OneHandedWeapon",
-    "price": 5384,
-    "weight": 1.82,
-    "tier": 8.632379,
+    "price": 3517,
+    "weight": 1.92,
+    "tier": 6.763587,
```

Comment:
speed reduction is OK but should be offset by an increase in damage to match tiers